docs/customization.md

```diff
@@ -1,38 +1,53 @@
 ## Customization
-### Modifying the client library format
 
-Don't like the default client syntax?  Want a different language supported?  No problem!  OpenAPI Generator processes mustache templates with the [jmustache](https://github.com/samskivert/jmustache) engine.  You can modify our templates or make your own.
+### Modifying a template
 
-You can look at `modules/openapi-generator/src/main/resources/${your-language}` for examples. To make your own templates, create your own files and use the `-t` flag to specify your template folder.  It actually is that easy.
+Clone OpenAPI Generator and take a look at the following directory: `modules/openapi-generator/src/main/resources/${template}`. In here you'll see all of the generators available, for most pogramming languages, web application frameworks and web servers.
```

Comment:
minor typo: pogramming => programming

Reply:
I was talking about pogs! 
![image](https://user-images.githubusercontent.com/67381/41252530-6533bd80-6d8b-11e8-82a3-0889abbcd5be.png)

🤣 fixed it! Thank you.